db/migrations/0004_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.0.2 on 2024-11-05 16:53
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_moviesession_cinema_hall_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='movies_actor', to='db.Actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movies_genre', to='db.Genre'),
```

Comment:
The `related_name='movies_genre'` is inconsistent with typical naming conventions. Consider using `related_name='movies'` to reflect the relationship from `Genre` to `Movie` more clearly.